Sources/OBCalendar/Model/CalendarModel.swift

```diff
@@ -27,14 +27,71 @@ public enum CalendarModel {
             case currentMonth
         }
         
-        public enum RangeType {
+        public enum RangeType: Equatable {
             case outOfRange(DateType)
             case insideRange(DateType)
+            
+            var dateType: DateType {
+                switch self {
+                case .outOfRange(let dateType):
```

Comment:
Can we change here like: `  case .outOfRange(let dateType), .insideRange(let dateType):
            return dateType
        }`